a/activemq/Dockerfiles/latest_centos_7/build/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ppc64le/centos:7
```

Comment:
agree with Gerrit on using ubi 8 instead of centos:7 unless of course there is an explicit need for centos:7 and I believe the image should come with a multiarch manifest, so we may not need the "ppc64le/" part explicitly

Reply:
Regarding selecting centos7, I agree with @gerrith3 and @seth-priya. However, ActiveMQ builds/Run requires java-7 https://activemq.apache.org/version-5-getting-started.html#installation-procedure-for-unix. I'll try ubi8, not sure if it supports JAVA 7.

Reply:
regarding the license file, there was already a one at the root https://github.com/ppc64le/build-scripts/tree/master/a/activemq